services/order.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket_data in tickets:
+            movie_session = ticket_data.get("movie_session")
+            row = ticket_data.get("row")
+            seat = ticket_data.get("seat")
+
+            ticket = Ticket(
+                movie_session_id=movie_session,
+                order=order,
+                row=row,
+                seat=seat
+            )
+
+            ticket.full_clean()
+            ticket.save()
+
+        return order
+
+
+def get_orders(username: str = None) -> Order:
+    if username:
+        user = get_user_model().objects.get(username=username)
```

Comment:
The `get` method can raise a `DoesNotExist` exception if the user with the given `username` is not found. Consider using `get_object_or_404` or handling the exception to prevent potential crashes.